apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
@dannykng does it make sense to have this as a Realtime guide?

Reply:
Enough content here to make it a separate thing. 

Plus people will see it in the side nav and hopefully realize they should read it.